kuka_kr270_support/urdf/kr270r2700_macro.xacro

```diff
@@ -0,0 +1,181 @@
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://kuka_kr270_support/meshes/kr270r2700/collision/link_1.stl"/>
+        </geometry>
+      </collision>
+    </link>
+    <link name="${prefix}link_2">
+      <visual>
+        <origin xyz="0 0 0" rpy="0 ${radians(90)} 0"/>
+        <geometry>
+          <mesh filename="package://kuka_kr270_support/meshes/kr270r2700/visual/link_2.dae"/>
+        </geometry>
+        <xacro:material_kuka_orange />
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 ${radians(90)} 0"/>
+        <geometry>
+          <mesh filename="package://kuka_kr270_support/meshes/kr270r2700/collision/link_2.stl"/>
+        </geometry>
+      </collision>
+    </link>
+    <link name="${prefix}link_3">
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://kuka_kr270_support/meshes/kr270r2700/visual/link_3.dae"/>
+        </geometry>
+        <xacro:material_kuka_orange />
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://kuka_kr270_support/meshes/kr270r2700/collision/link_3.stl"/>
+        </geometry>
+      </collision>
+    </link>
+    <link name="${prefix}link_4">
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://kuka_kr270_support/meshes/kr270r2700/visual/link_4.dae"/>
+        </geometry>
+        <xacro:material_kuka_orange />
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://kuka_kr270_support/meshes/kr270r2700/collision/link_4.stl"/>
+        </geometry>
+      </collision>
+    </link>
+    <link name="${prefix}link_5">
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://kuka_kr270_support/meshes/kr270r2700/visual/link_5.dae"/>
+        </geometry>
+        <xacro:material_kuka_orange />
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://kuka_kr270_support/meshes/kr270r2700/collision/link_5.stl"/>
+        </geometry>
+      </collision>
+    </link>
+    <link name="${prefix}link_6">
+      <visual>
+        <origin xyz="0.240 0 0 " rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://kuka_kr270_support/meshes/kr270r2700/visual/link_6.dae"/>
+        </geometry>
+        <xacro:material_kuka_black />
+      </visual>
+      <collision>
+        <origin xyz="0.240 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://kuka_kr270_support/meshes/kr270r2700/collision/link_6.stl"/>
+        </geometry>
+      </collision>
+    </link>
+    <!-- This frame corresponds to the $TOOL coordinate system in KUKA KRC controllers. -->
+    <link name="${prefix}flange"/>
```

Comment:
Please move this to where the other ROS-Industrial specific `link`s are defined.

Similar to how this is done for `tool0` and `base`.

Reply:
See 05382e115017b9931e28c52324f3fc90b2948639